index.html

```diff
@@ -5081,19 +5083,17 @@ <h3>Signature-Based Correlation</h3>
 
         <p>
 The contents of a [=verifiable credential=] are secured using a [=securing
-mechanism=]. Values used to represent the securing mechanism create a greater
-risk of correlation when the same values are used across more than one session
-or domain and the value does not change. Examples of these sorts of values
-include the following:
+mechanism=]. Values representing the securing mechanism pose a greater
+risk of correlation when they remain the same across multiple sessions
+or domains. Examples of these include the following values:
         </p>
 
         <ul>
           <li>
-the binary value of the digital signature,
+the binary value of the digital signature
           </li>
           <li>
-timestamp information associated with when the digital signature was created,
-and
+timestamp information associated with the creation of the digital signature
           </li>
           <li>
 cryptographic material, such as a public key identifier, associated with the
```

Comment:
[Again](https://github.com/w3c/vc-data-model/pull/1554/files#r1731438266). Replace 5099 with the two lines in this suggestion, and then delete what will then be redundant ("`digital signature.`") in line 5101.
```suggestion
cryptographic material associated with the digital signature, such as
a public key identifier
```